### PR TITLE
testing ci build info

### PR DESCRIPTION
**Pull request by roman01la**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -55,3 +79,11 @@ workflows:
...
     jobs:
       - build
+      - create-release:
+          requires:
+            - build
+          filters:
+            branches:
+              only:
+                - master
+                - roman/test-release
```
Comment (DReigada) [inline .circleci/config.yml:89]: Let's remove this before merging it

**Comment by accounts-pitch-io** [comment]
Potato.

**Comment by accounts-pitch-io** [comment]
UIx 534ms 1.5x
Reagent 1017ms 2.8x


UIx 533ms 1.6x
Reagent 1027ms 3.1x

**Comment by accounts-pitch-io** [comment]
master:
React 349ms
UIx 563ms 1.6x
Reagent 955ms 2.7x


PR:
React 335ms
UIx 533ms 1.6x
Reagent 1027ms 3.1x

**Comment by accounts-pitch-io** [comment]
master:
React 3257ms
UIx 5577ms 1.7x
Reagent 10018ms 3.1x


PR:
React 335ms
UIx 533ms 1.6x
Reagent 1027ms 3.1x

**Comment by accounts-pitch-io** [comment]
master:
React 335ms
UIx 533ms 1.6x
Reagent 1027ms 3.1x


PR:
React 3270ms
UIx 5315ms 1.6x
Reagent 9549ms 2.9x

**Comment by accounts-pitch-io** [comment]
master:
React 335ms
UIx 533ms 1.6x
Reagent 1027ms 3.1x


PR:
React 3701ms
UIx 5198ms 1.4x
Reagent 9298ms 2.5x

**Comment by accounts-pitch-io** [comment]
master:
<Error><Code>NotFound</Code><Message>NotFound: Not Found&#xA;&#x9;status code: 404, request id: BKBPE1B5V0BJWDMF, host id: VYdNRLx1BU6DfU2s5N1wyCclWSExY/PG+XfDDVDW04GbQAeyup4EnsjgeNld7SFa2+YosL2Wo3I=</Message><Resource>github-production-release-asset-2e65be402790508/f0a76f78-b974-4e30-8a36-ad9c858be007</Resource></Error>

PR:
React 3827ms
UIx 6335ms 1.7x
Reagent 11409ms 3x

**Pull request merged**